packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -243,18 +262,29 @@ export class ElectronMainApplication {
                 // Issue: https://github.com/eclipse-theia/theia/issues/8577
                 nodeIntegrationInWorker: false,
             },
-            ...windowOptionsFromConfig,
+            ...this.config.electron?.windowOptions || {},
+        };
+    }
+
+    protected getDefaultWindowOptions(): TheiaBrowserWindowOptions {
+        return {
+            isFullScreen: false,
+            isMaximized: false,
```

Comment:
Should we add this to `getDefaultOptions` instead?

Reply:
No, I remove it from there because it will set these values to the options of the last window